### DISCUSSION TITLE
Shuffler notes

### DISCUSSION THREAD

**Discussion by sakkammadam** [sakkammadam/scratchStuff#7] [Announcements]
@hbaird7877 , @abejr17 - 
 Team - 
 This is to let you know that I have completed the Shuffler implementation. 
 I have introduced Shuffler.h and Shuffler.cpp to the project.
 Shuffler will essentially sort and count all instances of a token occurring within a partition file.
 From an output standpoint - 

![Screenshot from 2022-10-21 01-09-36](https://user-images.githubusercontent.com/10201590/197146580-8acc533f-f61b-4e5a-ae54-cad76adf7af3.png)

There will be a temp_shuffler folder within the input folder containing files

Just like previously, within the temp_shuffler folder, you will find sub-folders named after each file

![Screenshot from 2022-10-21 01-09-48](https://user-images.githubusercontent.com/10201590/197146757-e4addc23-f93a-4930-8be6-86b78ef8a754.png)

On clicking a sub-folder - you will observe the same partition style splits.

![Screenshot from 2022-10-21 01-10-17](https://user-images.githubusercontent.com/10201590/197146877-f313ab38-844f-4c42-9682-b7825b038ba1.png)

This time though, you will observe the file size has reduced and we are now counting the token occurrences.

Please take a moment to look at the code and let me know if you have any questions/concerns.

Thanks,
Sooraj

**Comment by hbaird7877** [comment]
I cannot access the source files, but everything I could see looked correct.  Thanks for all of your efforts on this.  Best, Hal.
